pages/developers.tsx

```diff
@@ -101,6 +101,15 @@ export default function PageDevelopers() {
                   defaultValue={filtersByIssueState[0]}
                   options={filtersByIssueState}
                   onChange={handleChangeFilterByState}
+                  styles={{
```

Comment:
this can easily be done via scss without having to resort to javascript-styles, mainly because we already mutilating the styles of this component